src/sections/about.html

```diff
@@ -0,0 +1,16 @@
+<div>
+  <p class="font-body font-normal text-black text-2xl lg:text-4xl md:text-3xl px-8 md:px-16 lg:px-24 ">
+    We are a Berlin-based communications agency specialising in helping
+    companies and brands find their voice and position themselves. Whether it's
+    brand building, identity sharpening, or communication strategy, we bring a
+    fresh perspective to established structures, develop tailor-made
+    communication solutions, and ensure that messages resonate. With over a
+    decade of experience, we have honed our skills in giving brands their
+    unmistakable identity and making them visible in the long term.
+  </p>
+  <div class="flex justify-center mt-16 items-center sticky top-0">
```

Comment:
This image is too small 
![group 5  reveiws](https://github.com/DevRamona/the_Baumesiter/assets/78850411/51b733a9-7c76-44c5-a37d-7de3ed8ae90e) check the figma and compare to your UI.